src/component/InputBook/InputBook.module.css

```diff
@@ -0,0 +1,71 @@
+.line {
+  width: 93rem;
+  height: -0.875rem;
+  margin: 2.5rem 3rem;
+  border: solid 1px #e8e8e8;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: var(--warm-grey);
+}
+
+.LessonPanel {
+  width: 13.5rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 35.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.LessonPanel2 {
+  width: 17.813rem;
+  height: 1.813rem;
+  margin: 1.188rem 2.125rem 0;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.RectangleButton {
+  width: 11.5rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
+
+.ADDBOOK {
+  width: 8.938rem;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+footer {
+  padding: 0 95px;
+  height: 291px;
+}
```

Comment:
- [ ] The footer should be aligned in the same line as shown in the design. Please make changes to it
###  Figma design
![figma](https://user-images.githubusercontent.com/81727988/174356797-bd764b62-0c10-4517-a64d-e37dcf91ca50.png)
### Your design

![your](https://user-images.githubusercontent.com/81727988/174356979-e0a2d8d2-8ce0-4b4f-a712-a8644de23f29.png)